src/test/java/net/kaczmarzyk/e2e/converter/DateE2eTest.java

```diff
@@ -108,4 +125,42 @@ public void findsByDateAfter() throws Exception {
 			.andExpect(jsonPath("$[2].firstName").value("Minnie"))
 			.andExpect(jsonPath("$[3]").doesNotExist());
 	}
+
+	@Test
+	public void findsByEqualDayUsingDefaultConfig() throws Exception {
+		customer("Barry", "Benson")
+				.registrationDate(2014, 3, 16)
+				.build(em);
+		customer("Adam", "Flayman")
+				.registrationDate(2014, 3, 14, 23, 59, 59, 999)
+				.build(em);
+
+		mockMvc.perform(get("/customers")
+						.param("registeredDayEqual", "2014-03-15")
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isOk())
+				.andExpect(jsonPath("$").isArray())
+				.andExpect(jsonPath("$[0].firstName").value("Homer"))
+				.andExpect(jsonPath("$[1].firstName").value("Moe"))
+				.andExpect(jsonPath("$[2]").doesNotExist());
+	}
+
+	@Test
+	public void findsByEqualDayUsingCustomConfigIgnoringTime() throws Exception {
+		customer("Barry", "Benson")
+				.registrationDate(2014, 3, 16)
+				.build(em);
+		customer("Adam", "Flayman")
+				.registrationDate(2014, 3, 14, 23, 59, 59, 999)
+				.build(em);
+
+		mockMvc.perform(get("/customers")
+						.param("registeredDayEqualCustomConfig", "2014-03-15T12:34:19")
+						.accept(MediaType.APPLICATION_JSON))
+				.andExpect(status().isOk())
+				.andExpect(jsonPath("$").isArray())
+				.andExpect(jsonPath("$[0].firstName").value("Homer"))
+				.andExpect(jsonPath("$[1].firstName").value("Moe"))
```

Comment:
see the previous comment, probably worth adding more data here as well for better readability and to make sure that this test does its purpose even if integration test base is modified